README.md

```diff
@@ -1,14 +1,332 @@
+   2. `call` syntax mirrors [near-cli](https://github.com/near/near-cli) and either returns the successful return value of the given function or throws the encountered error. If you want to inspect a full transaction and/or avoid the `throw` behavior, you can use `call_raw` instead.
+   3. While `call` is invoked on the account _doing the call_ (`alice.call(contract, …)`), `view` is invoked on the account _being viewed_ (`contract.view(…)`). This is because the caller of a view is irrelevant and ignored.
+   4. Gotcha: the full account names may or may not match the strings passed to `createAccount` and `createAndDeploy`, which is why you must write `alice.call(contract, …)` rather than `alice.call('contract-account-name', …)`. But! The `Account` class overrides `toJSON` so that you can pass `{account_id: alice}` in arguments rather than `{account_id: alice.accountId}`. If you need the generated account ID in some other circumstance, remember to use `alice.accountId`.
+
+
+See the [tests](https://github.com/near/workspaces-js/tree/main/__tests__) directory in this project for more examples.
+
+Quick Start with AVA
+===========
+Since `near-workspaces` is designed for concurrency, AVA is a great fit, because it runs tests concurrently by default. To use`NEAR Workspaces` with AVA:
+ 1. Start with the basic setup described [here](https://github.com/avajs/ava).
+ 2. Add custom script for running tests on Testnet (if needed). Check instructions in `Running on Testnet` section.
+ 3. Add your tests following these example:
+
+  ```ts
+  import {Workspace} from 'near-workspaces';
+  import anyTest, {TestFn} from 'ava'
+
+  const test = anyTest as TestFn<{workspace: Workspace}>;
+  test.before(async t => {
+    t.context.workspace = await Workspace.init(async ({root}) => ({
+      contract: await root.createAndDeploy(
+        'account-id-for-contract',
+        'path/to/contract/file.wasm',
+      ),
+      /* Account that you will be able to use in your tests */
+      ali: await root.createAccount('ali'),
+    }));
+  })
+
+  test('Test name', async t => {
+    /* Each test is making a "fork", a copy of the
+    workspace, that was created in "before" function.
+    It allows you to isolate each test and run them concurrently */
+    await t.context.workspace.fork(async ({contract, ali}) => {
+      await ali.call(contract, 'set_status', {message: 'hello'});
+      const result: string = await contract.view('get_status', {
+        account_id: ali,
+      });
+      t.is(result, 'hello');
+    });
+  });
+  ```
+
+  For a working example, see [near-examples/rust-status-message/tests-ava](https://github.com/near-examples/rust-status-message/blob/master/tests-ava/__tests__/main.ava.ts).
```

Comment:
```suggestion
```

Reply:
This example uses the old workspaces-js, which API has become incompatible. I suggest update the example then point to it